### PR TITLE
Verification page

### PR DESCRIPTION
**Pull request by nabhan-au**
[Build Status](https://app.travis-ci.com/ThaiRepose/thairepose)
[codecov](https://codecov.io/gh/ThaiRepose/thairepose)
### What is the feature?
- Page for verification email
- Make verify email as an optional feature
### What is the solution?
- CSS for verification_sent.html and verified_email_resend.html
- Edit setting to make email verification is an optional requirement
### How to test?
1. Run tailwind start and runserver
2. Try to signup to verification email page
### Screen shot
image
image
image
image

(Include screenshots if any)

### CODE REVIEW

**Comment by tboonma** [comment]
I followed the instructions in README, so I got this error when I tried to signup.
Screen Shot 2564-11-03 at 14 19 37
Please provide more detail.

**Comment by tboonma** [comment]
If there's any error, the signup information shouldn't be recorded. I got the above error and the user has been recorded without sending verification email.

**Comment by nabhan-au** [comment]
This problem happen because email or username is wrong or It may happen because you use your email and not set your email to less secure

File: users/adapter.py [hunk 1]
```diff
@@ -28,8 +30,25 @@ def save_user(self, request, user, form, commit=True):
...
+        if 'user_email' in request.session:
+            del request.session['user_email']
+        request.session['user_email'] = user.email
         return user
 
+    def respond_email_verification_sent(self, request, user):
+        """Show email_verification_sent page with assign user_email variable to request.session.
+
+        Args:
+            user (Usser): user model
```
Comment (tboonma) [inline users/adapter.py:42]: A bit misspelling here.

**Review by tboonma — COMMENTED** [review]
1 misspelling in `users/adapter.py` docstring.

**Review by tboonma — APPROVED** [review]
Nice work. Good job bro!

**Pull request merged**